util/ConfigFileProcess.h

```diff
@@ -0,0 +1,22 @@
+#ifndef _ConfigFileProcess_h_
+#define _ConfigFileProcess_h_
+
+#include <vector>
+#include <string>
+#include <map>
+#include <iostream>
+
+#include "Directories.h"
+#include "OptionsDB.h"
+#include "i18n.h"
+#include "Version.h"
+#include "XMLDoc.h"
+
+#include <boost/filesystem/fstream.hpp>
+
+// client/human/chmain, server/dmain and client/AI/camain have common code
```

Comment:
@memem359 As the api docs are generated with doxygen, [Doxygen Manual - Documenting the code](http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html) might be helpful (use `@` style commands)